src/functional/program_processors/codegens/gtfn/codegen.py

```diff
@@ -124,7 +124,12 @@ def visit_OffsetLiteral(self, node: gtfn_ir.OffsetLiteral, **kwargs: Any) -> str
     )
 
     def visit_FunCall(self, node: gtfn_ir.FunCall, **kwargs):
-        return self.generic_visit(node, fun_name=self.visit(node.fun))
+        if isinstance(node.fun, gtfn_ir.SymRef) and node.fun.id in self.user_defined_function_ids:
+            fun_name = f"{self.visit(node.fun)}{{}}()"
+        else:
+            fun_name = self.visit(node.fun)
+
+        return self.generic_visit(node, fun_name=fun_name)
```

Comment:
These changes enable calling a user-defined function from a stencil, we didn't/don't have a test for this because we always inline.